flows/runs/event_log.go

```diff
@@ -0,0 +1,56 @@
+package runs
+
+import (
+	"encoding/json"
+
+	"github.com/nyaruka/goflow/flows"
+	"github.com/nyaruka/goflow/utils"
+)
+
+type eventLogEntry struct {
```

Comment:
so `session.Log() []flows.LogEntry`  ? It's a bit vague but I like the brevity

how do we feel about using `log` in response, e.g. 

```json
{
  "session": {},
  "log": [{}, {}]
}
```